bool left_rumble
bool right_rumble
bool front_rumble
float32 intensity
float32 delay